app/models/experimentation/experiment.rb

```diff
@@ -22,7 +22,7 @@ class Experiment < ActiveRecord::Base
     }, _prefix: true
 
     def self.candidate_patients
-      Patient.from(Patient.with_hypertension, :patients)
+      Patient.with_hypertension
```

Comment:
I think this was leftover from an earlier version of this query that was hacking around default scope things?

Reply:
It may be related to the phone number default scope. I recall the problem was a "distinct" that didn't play well with the rest of the query.